local-cli/server/runServer.js

```diff
@@ -83,7 +83,7 @@ function getPackagerServer(args, config) {
       'bmp', 'gif', 'jpg', 'jpeg', 'png', 'psd', 'svg', 'webp', // Image formats
       'm4v', 'mov', 'mp4', 'mpeg', 'mpg', 'webm', // Video formats
       'aac', 'aiff', 'caf', 'm4a', 'mp3', 'wav', // Audio formats
-      'html', // Document formats
+      'html', 'pdf', // Document formats
```

Comment:
I think `pdf` is one of these things that should work as is in WebView (otherwise, except `html` I don't see any other use-case for it in React Native)